client/block-builder/src/lib.rs

```diff
@@ -19,7 +19,7 @@
 //! Substrate block builder
 //!
 //! This crate provides the [`BlockBuilder`] utility and the corresponding runtime api
-//! [`BlockBuilder`](sp_block_builder::BlockBuilder).Error
+//! [`BlockBuilder`](./struct.BlockBuilder.html).Error
```

Comment:
```suggestion
//! [`BlockBuilder`](sp_block_builder::BlockBuilder).
```

The change was not correct

Reply:
OK, just learned about [intra-documentation links](https://rust-lang.github.io/rfcs/1946-intra-rustdoc-links.html).
I am now trying to see how I can support them in **cargo-unleash**'s [gen-readme command](https://github.com/paritytech/cargo-unleash/blob/master/src/commands/readme.rs).